ext/jenkins/beaker-tests.sh

```diff
@@ -19,14 +21,27 @@ export PUPPETDB_USE_PROXIES=false
 export BEAKER_project=PuppetDB
 export BEAKER_department=eso-dept
 export BEAKER_PRESERVE_HOSTS=onfail
+export BEAKER_COLLECT_PERF_DATA=aggressive
 
-# S3 params
-export PUPPETDB_PACKAGE_REPO_HOST="puppetdb-prerelease.s3.amazonaws.com"
-export PUPPETDB_PACKAGE_REPO_URL="http://puppetdb-prerelease.s3.amazonaws.com/puppetdb/${PACKAGE_BUILD_VERSION}"
-export PUPPETDB_PACKAGE_BUILD_VERSION=$PACKAGE_BUILD_VERSION
-export BEAKER_CONFIG=$LAYOUT
-export INSTALL_MODE=$INSTALL_TYPE
+# Transform LAYOUT from old static host config style to genconfig2 style
+# Once the necessary PRs (puppetlabs/puppetdb#1636 and
+# puppetlabs/ci-job-configs#529) for both this repo and ci-job-configs have been
+# merged, this logic can be removed.
+[ "$LAYOUT" = "ec2-west-el7-64mda-el7-64a" ] \
```

Comment:
`LAYOUT` is still used by the beaker-hostgenerator to make the host file. This was initially to use in-conjunction with current ci-job-configs settings, but yeah we can rip this part out.